app/uk/gov/hmrc/exports/mongock/changesets/CacheChangelog.java

```diff
@@ -27,4 +33,27 @@
     @ChangeSet(order = "001", id = "Exports DB Baseline", author = "Paulo Monteiro")
     public void dbBaseline(MongoDatabase db) {
     }
+
+    @ChangeSet(order = "002", id = "CEDS-2231 Change country name to country code for location page", author = "Patryk Rudnicki")
+    public void updateAllCountriesNameToCodesForLocationPage(MongoDatabase db) {
+        Document query = new Document();
+        CountriesService service = new CountriesService();
+
+        FindIterable<Document> documents = db.getCollection(collection).find(new BasicDBObject(query));
+
+        for (Document document : documents) {
```

Comment:
Just to check I'm reading this right.    
You are looking for existing declarations with a country name in location>goodsLocation>country converting that country name to code and reseting it back on the same country field?

Reply:
1) I'm going through all existing declarations
2) Checking if country field is defined
3) If it's defined I'm retrieving existing country name
4) Based on the country name I'm generating country code
5) Replacing old value in the model from country name to country code
6) Replacing whole declaration with updated on inside the cache